site.yml

```
@@ -64,7 +64,7 @@ asciidoc:
     php-supported-versions-url: https://secure.php.net/supported-versions.php
     http-status-codes-base-url: https://developer.mozilla.org/en-US/docs/Web/Status
     minimum-php-version: 7.1
```

Do you also need to do:
```suggestion
    minimum-php-version: '7.1'
```
similar to #2453

---

This isn't required. It's only when a value, that can be interpreted as a number, not a string, is used and it ends with `0`. I'm sure that I saw an explicit reference to this in the docs, but I'm unable to find it at the moment, in either the AsciiDoc User Manual or the Antora docs.

---

I just asked in the asciidoctor forum, and hope to find out where the reference is (or contribute it if it's not available).